File: uri/5 - Mathematics/2170/Main.java
import java.util.*;

public class Main{

    static double percentual(double x, double y){
      return (y / x - 1)  * 100;
    } 

    public static void main(String args[]){
        Scanner scanner  = new Scanner(System.in);
        double a,b;
        int i=1;

        while(scanner.hasNext()){
          a = scanner.nextDouble();
          b = scanner.nextDouble();
          System.out.format("Projeto %d:\n", i++);
          System.out.format("Percentual dos juros da aplicacao: %.2f %%\n\n", percentual(a,b) );
        }
    }
}

File: uri/5 - Mathematics/2170/Main.c
#include <stdio.h>

// #define PERCENTUAL(x,y) ((y - x) / (float)x) * 100
#define PERCENTUAL(x,y) (y / x - 1)  * 100

int main(){

    double x,y;
    long long i = 1;
    while( scanf("%lf %lf", &x, &y) != EOF ){
      printf("Projeto %lld:\n", i++);
      printf("Percentual dos juros da aplicacao: %.2f %%\n\n", PERCENTUAL(x,y) );
    }

    return 0;
}